src/include/platform/ConfigurationManager.h

```diff
@@ -79,6 +79,13 @@ class ConfigurationManager
     CHIP_ERROR GetDeviceCertificate(uint8_t * buf, size_t bufSize, size_t & certLen);
     CHIP_ERROR GetDeviceIntermediateCACerts(uint8_t * buf, size_t bufSize, size_t & certsLen);
     CHIP_ERROR GetDevicePrivateKey(uint8_t * buf, size_t bufSize, size_t & keyLen);
+    CHIP_ERROR GetDeviceDACCertificate(uint8_t * buf, size_t bufSize, size_t & dacLen);
+    CHIP_ERROR GetDevicePAICertificate(uint8_t * buf, size_t bufSize, size_t & paiLen);
+    CHIP_ERROR GetDevicePAACertificate(const uint8_t * skid, size_t skidLen, uint8_t * buf, size_t bufSize, size_t & paaLen);
+    CHIP_ERROR GetDeviceCertificationDeclaration(uint8_t * buf, size_t bufSize, size_t & certDeclLen);
+    CHIP_ERROR GetDeviceAttestationSignature(const uint8_t * attestationElements, size_t attestationElementsSize,
+                                             const uint8_t * attestationChallenge, size_t attestationChallengeSize, uint8_t * buf,
+                                             size_t bufSize, size_t & signatureLen);
```

Comment:
Again, use spans for input byte buffers.  And please document that this is implementing the part of the spec that signs the attestation message (NOT attestation elements!) and attestation challenge with the device attestation private key.  And that that last bit is why it's on the configuration manager.